pkg/mcproducer/templates/replace-kernel-module.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+kmm_config_file_filepath="$WORKER_CONFIG_FILEPATH"
+kernel_module_image_filepath="$KERNEL_MODULE_IMAGE_FILEPATH"
+in_tree_module_to_remove="$IN_TREE_MODULE_TO_REMOVE"
+kernel_module="$KERNEL_MODULE"
+worker_image="$WORKER_IMAGE"
+
+create_kmm_config() {
+    # Write YAML content to the file
+    cat <<EOF > "$kmm_config_file_filepath"
+containerImage: $kernel_module_image_filepath
+inTreeModuleToRemove: $in_tree_module_to_remove
```

Comment:
I think this variable is not mandatory, so we should quote it to prevent the YAML from being unparsable if `$in_tree_module_to_remove` is an empty string.

Reply:
actually, yaml does not become unparsable, i have verified it. This way i don't need to have additional ifs in bash script.

Reply:
TIL.